harness/tests/experiment/keras/test_keras_data.py

```diff
@@ -174,7 +171,41 @@ def test_enqueuer(
     with keras._build_enqueuer(
         sequence=IdentitySequence(100),
         workers=workers,
-        use_multiprocessing=use_multiprocessing,
+        use_multiprocessing=False,
+        max_queue_size=10,
+        shard_rank=rank,
+        num_shards=size,
+        repeat=False,
+        shuffle=shuffle,
+        shuffle_seed=777,
+        prior_batches_trained=skip,
+    ) as enqueuer:
+        assert list(enqueuer.data()) == list(sampler.yield_epoch()), "first epoch was wrong"
+        assert list(enqueuer.data()) == list(sampler.yield_epoch()), "second epoch was wrong"
+        assert list(enqueuer.data()) == list(sampler.yield_epoch()), "third epoch was wrong"
+
+
+@pytest.mark.parametrize("workers", [1, 5])
+def test_enqueuer_multiprocessing(workers: int) -> None:
+    """Same enqueuer test as above, but with multiprocessing enabled.
+
+    A single, arbitrary set of parameters from the above test is chosen in order to exercise the
```

Comment:
This docstring needs a slight update, but otherwise LGTM

Reply:
Ooh. Thanks for catching that.